### PR TITLE
added get_rootfs to oci

### PR DESCRIPTION
**Pull request by Mossaka**

### CODE REVIEW

File: crates/containerd-shim-wasmtime-v1/src/sandbox/oci.rs [hunk 1]
```diff
@@ -28,6 +30,17 @@ pub fn load(path: &str) -> Result<Spec, Error> {
     Ok(spec)
 }
 
+pub fn get_root(spec: &Spec) -> Result<&PathBuf, Error> {
+    let root = spec.root().as_ref().unwrap();
```
Comment (cpuguy83) [inline crates/containerd-shim-wasmtime-v1/src/sandbox/oci.rs:34]: Doesn't seem like we are using the error return in this function.

File: crates/containerd-shim-wasmtime-v1/src/sandbox/oci.rs [hunk 2]
```diff
@@ -4,10 +4,12 @@ use oci_spec::runtime::Spec;
 use oci_spec::OciSpecError;
 use serde_json as json;
 use std::fs::{File, OpenOptions};
-use std::path::Path;
+use std::path::{Path};
+use cap_std::path::PathBuf;
 use thiserror::Error;
 use wasmtime_wasi::sync::file::File as WasiFile;
 use wasmtime_wasi::{Dir as WasiDir, WasiCtxBuilder};
+use log::info;
```
Comment (cpuguy83) [inline crates/containerd-shim-wasmtime-v1/src/sandbox/oci.rs:12]: Looks unused

File: crates/containerd-shim-wasmtime-v1/src/sandbox/oci.rs (hunk 1 above)
Comment (Mossaka) [reply crates/containerd-shim-wasmtime-v1/src/sandbox/oci.rs:33]: fixed

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**